fitting.qmd

```diff
@@ -0,0 +1,624 @@
+---
+title: "Fitting odin models with monty"
+format:
+  revealjs:
+    preview-links: auto
+    footer: "[mrc-ide/odin-monty](.)"
+execute:
+  echo: true
+  message: true
+  output: true
+  warning: true
+---
+
+```{r}
+#| include: false
+#| cache: false
+set.seed(1)
+source("common.R")
+options(monty.progress = FALSE)
+```
+# A pragmatic introduction
+
+```{r}
+library(odin2)
+library(dust2)
+library(monty)
+```
+
+## Previously, on "Introduction to odin"
+
+* We created some simple compartmental models
+* We ran these and observed trajectories over time
+* We saw that stochastic models produce a family of trajectories
+
+## Our model {.smaller}
+
+```{r}
+#| echo: false
+#| results: "asis"
+r_output(readLines("models/sir-basic.R"))
+```
+
+## Example output {.smaller}
+
+```{r}
+#| include: false
+sir <- odin("models/sir-basic.R")
+```
+
+```{r}
+sys <- dust_system_create(sir, n_particles = 30)
+dust_system_set_state_initial(sys)
+t <- seq(0, 100)
+y <- dust_system_simulate(sys, t)
+incidence <- dust_unpack_state(sys, y)$incidence
```

Comment:
Fails here as incidence is not in the model output - do you mean to plot `I` instead?